src/repositories/friendship.repository.ts

```diff
@@ -22,6 +26,17 @@ export class FriendshipRepository extends Repository {
     return body.friendship_statuses;
   }
 
+  async findFollowers(userId: string | number, userNameToLookFor: string) {
+    if (!userNameToLookFor) {
+      return [];
+    }
+
+    const { body } = await this.client.request.send<FriendshipRepositoryFindFollowerResponseRootObject>({
+      url: `/api/v1/friendships/${userId}/followers?query=${userNameToLookFor}`,
```

Comment:
Just add the `query` param in the [followers-feed](https://github.com/dilame/instagram-private-api/blob/master/src/feeds/account-followers.feed.ts), why create another method with the same URL?